DatadogSessionReplay/Sources/Processor/SRDataModelsBuilder/WireframesBuilder.swift

```diff
@@ -192,6 +192,8 @@ internal class WireframesBuilder {
     }
 }
 
+internal typealias WireframesBuilder = SessionReplayWireframesBuilder
```

Comment:
worth adding note that it's done to have more unique name that's exposed through public-internal access level.